### ISSUE TITLE
Cannot load locales when inside a Workspace

### ISSUE THREAD

**Issue by francisbr** [Baptistemontan/leptos_i18n#53]
I get: `error: Could not found "[package.metadata.leptos-i18n]" in cargo manifest (Cargo.toml)`, when calling `leptos_i18n::load_locales!();`. I think there's a problem when trying to load the locales when inside a worspace? Or maybe I just missunderstood something and am placing my metadata in the wrong place?

Here is my folder structure:
```
frontend/
├─ api_sdk/
│  ├─ .../
├─ app/
│  ├─ src/
│  │  ├─ main.rs
│  │  ├─ ...
│  ├─ translations/
│  ├─ Cargo.toml
│  ├─ index.html
├─ Cargo.toml
```
The `app/` folder is where is my leptos code code and I am trying to use leptos_i18n. Here is my setup:
```toml
# frontend/Cargo.toml
[workspace]
resolver = "2"
members = ["app", "api_sdk"]
default-members = ["app"]
```

```toml
# frontend/app/Cargo.toml
[package]
name = "app"
version = "0.1.0"
edition = "2021"

[package.metadata.leptos-i18n]
default = "en"
locales = ["en", "fr"]
locales-dir = "./translations"

[dependencies.leptos]
version = "0.5.0-rc1"
features = ["csr", "tracing"]

[dependencies.leptos_i18n]
version = "0.2.0-beta5"
```

And so I am trying to call `leptos_i18n::load_locales!();` inside `frontend/app/src/main.rs` and I get the error specified.

**Comment by Baptistemontan** [comment]
I think I went monkey mode and just did `./Cargo.toml` for the path of the manifest without considering workspaces, I need to find a better way, I think there is a `CARGO_MANIFEST` env variable for that but I have to look into it. 
For now I think if you put the metadata part in `frontend/Cargo.toml` it should work, you will need to change `locales-dir` to `"./app/translations"` too.

**Comment by francisbr** [comment]
I might be totally wrong on this one, but I don't think I can do that? If I want to move the metadata part in `frontend/Cargo.toml` I have to either:

1. Put it in `[workspace.package.metadata.leptos-i18n]`
2. Or `[workspace.metadata.leptos-i18n]`

In the first option, cargo complains that since I have juste declared a package I need a `main.rs` or `lib.rs`, which I don't have.

In the second one, I still get the same error, "package.metadata.leptos-i18n not found" which I guess makes sense since package.metadata != workspace.metadata?

_Btw I did try to do it and it didn't work. But maybe I made a mistake still not a pro rust dev :)_

**Comment by Baptistemontan** [comment]
No it make sense for Rust to complain, it's a bad job from me... 
I'm working on it and will release a new version when it's ready, I'll close the issue with the merge

**Comment by Baptistemontan** [comment]
So the env variable was `CARGO_MANIFEST_DIR` and it gives the package dir, I'm pushing it and waiting for CI but it should be good

**Comment by Baptistemontan** [comment]
Well Leptos just released `v0.5.0-rc3` so I will update to it before releasing `v0.2.0-beta6`

**Comment by Baptistemontan** [comment]
The release of Leptos `v0.5.0-rc3` broke the CI so it's taking longer than expected haha

**Comment by Baptistemontan** [comment]
Released the fix in `v0.2.0-beta6`, you should be able to update and make it work, feel free to open any new issue if another problem emerge!
